enterprise_subsidy/apps/subsidy/migrations/0017_auto_20230905_1511.py

```diff
@@ -0,0 +1,47 @@
+# Generated by Django 3.2.20 on 2023-09-05 15:11
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('subsidy', '0016_sf_opp_line_item_20230404_1656'),
+    ]
+
+    operations = [
+        migrations.AlterModelOptions(
+            name='subsidy',
+            options={'ordering': ['-created']},
+        ),
+        migrations.AddField(
+            model_name='historicalsubsidy',
+            name='is_hidden',
+            field=models.BooleanField(default=False, help_text='If set, this subsidy will be considered as soft-deleted or deactivated.'),
+        ),
+        migrations.AddField(
```

Comment:
how about using `hidden_at` (a datetime) instead? this lets us know _when_ something was deleted.

Reply:
We have history on this model that would let us know that, so either way is more-or-less the same.  I think `is_hidden` is slightly more user-friendly.

Reply:
Nit: While we are nit picking... why `is_hidden` and not `is_deleted`? For me this falls under something like be explicit in your naming and say what you are actually going to do. Are we hiding here or deleting? I think that deleting makes what is happening more obvious to later readers/users of the code.

Reply:
"hidden" is the best name we came up with at the time of grooming and ticket-writing.  `is_deleted` or `deleted_at` both work equally fine, but I think we're bike-shedding.